docs/samples/git-setup/policytemplates/version_4.Y.Z/sno-ran-full-du-profile/sno-ran-full-du-pg-v4-Y-Z-v1.yaml

```diff
@@ -0,0 +1,349 @@
+    - path: source-crs/AcceleratorsNS.yaml
+    - path: source-crs/AcceleratorsOperGroup.yaml
+    - path: source-crs/AcceleratorsSubscription.yaml
+      patches:
+      - spec:
+          channel: "stable"
+          source: certified-operators
+          installPlanApproval:
+            '{{hub $configMap:=(lookup "v1" "ConfigMap" "" (printf "%s-pg" .ManagedClusterName)) hub}}{{hub dig "data" "install-plan-approval" "Manual" $configMap hub}}'
+    - path: source-crs/AcceleratorsOperatorStatus.yaml
+    # LCA
+    - path: source-crs/LcaSubscriptionNS.yaml
+    - path: source-crs/LcaSubscriptionOperGroup.yaml
+    - path: source-crs/LcaSubscription.yaml
+      patches:
+      - spec:
+          source: redhat-operators
+          installPlanApproval:
+            '{{hub $configMap:=(lookup "v1" "ConfigMap" "" (printf "%s-pg" .ManagedClusterName)) hub}}{{hub dig "data" "install-plan-approval" "Manual" $configMap hub}}'
+    - path: source-crs/LcaSubscriptionOperGroup.yaml
+    # OADP
+    - path: source-crs/OadpSubscriptionNS.yaml
+    - path: source-crs/OadpSubscriptionOperGroup.yaml
+    - path: source-crs/OadpSubscription.yaml
+      patches:
+      - spec:
+          source: redhat-operators
+          installPlanApproval:
+            '{{hub $configMap:=(lookup "v1" "ConfigMap" "" (printf "%s-pg" .ManagedClusterName)) hub}}{{hub dig "data" "install-plan-approval" "Manual" $configMap hub}}'
+    - path: source-crs/OadpOperatorStatus.yaml
+    # Local storage operator
+    - path: source-crs/StorageNS.yaml
+    - path: source-crs/StorageOperGroup.yaml
+    - path: source-crs/StorageSubscription.yaml
+      patches:
+      - spec:
+          source: redhat-operators
+          installPlanApproval:
+            '{{hub $configMap:=(lookup "v1" "ConfigMap" "" (printf "%s-pg" .ManagedClusterName)) hub}}{{hub dig "data" "install-plan-approval" "Manual" $configMap hub}}'
+    - path: source-crs/StorageOperatorStatus.yaml
+# CONFIGURATION POLICY
+- name: v1-config-policy
+  manifests:
+    - path: source-crs/DisableOLMPprof.yaml
+    - path: source-crs/DisableSnoNetworkDiag.yaml
+    # REDUCE FOOTPRINT
+    - path: source-crs/ReduceMonitoringFootprint.yaml
+      patches:
+      - data:
+          config.yaml: |
+            alertmanagerMain:
+              enabled: false
+            nodeExporter:
+              collectors:
+                buddyinfo: {}
+                cpufreq: {}
+                ksmd: {}
+                mountstats: {}
+                netclass: {}
+                netdev: {}
+                processes: {}
+                systemd: {}
+                tcpstat: {}
+            prometheusK8s:
+              additionalAlertmanagerConfigs:
+              - apiVersion: v2
+                bearerToken:
+                  key: token
+                  name: observability-alertmanager-accessor
+                scheme: https
+                staticConfigs:
+                - '{{hub fromConfigMap "" "acm-route" "route" hub}}'
```

Comment:
Note, this configmap does not exist on the hub by default and would need to be created either manually or by a policy which copies the url from the appropriate route.